webrtc.html

```diff
@@ -6476,6 +6484,36 @@ <h2>
               </dl>
             </section>
           </div>
+          <div>
+            <pre class="idl">dictionary RTCLocalIceCandidateInit : RTCIceCandidateInit {
+  RTCIceServerTransportProtocol? relayProtocol = null;
+  USVString? url = null;
```

Comment:
The only excuse I can find for making these default to null is that we already did that mistake with the other parts of RTCIceCandidateInit.

Reply:
The dictionary defaults are driven by the defaults on the corresponding read-only attributes on the `RTCIceCandidate` interface, which predate this PR, even for [relayProtocol](https://w3c.github.io/webrtc-pc/#dom-rtcicecandidate-relayprotocol) and [url](https://w3c.github.io/webrtc-pc/#dom-rtcicecandidate-url)).